proposals/NNNN-Addition-of-a-way-to-notify-HMI-of-the-required-support-for-apps.md

```diff
@@ -0,0 +1,103 @@
+# Addition of the way to notify HMI of the required support for apps
+
+* Proposal: [SDL-NNNN](NNNN-Addition-of-a-way-to-notify-HMI-of-the-required-support-for-apps.md)
+* Author: [Akihiro Miyazaki (Nexty)](https://github.com/Akihiro-Miyazaki)
+* Status: **Awaiting review**
+* Impacted Platforms: [ Core / Java Suite / HMI / RPC / Protocol ]
+
+## Introduction
+This proposal is to provide a way to notify HMI of the required support for apps by using the new notification messages. 
+
+
+## Motivation
+If the system does not have something to support required for apps, apps may not work properly or be able to register. For example, in the case of Android OS, if a Media app connects via only USB, since the system cannot establish the audio path, Media app cannot play audio and does not register arbitrarily. At the result, it cannot allow user to know the reason why app cannot be registered. To solve these problems, we propose to add a new notification message to let HMI know the support the SDL app needs. By doing so, this will allow HMI to display the appropriate message and apps connectivity and operability will be improved.
+
+
+## Proposed solution
+To provide a way to notify HMI of the required support for apps, we will add new notification messages `Request Additional Support` and `Ping Apps` on Protocol spec and new RPCs `RequestAdditionalSupport` and `PingApps` on HMI_API.
+
+#### Protocol Spec
+The message `Request Additional Support` has two parameters `appName` and `requiredAdditionalSupport`. Also, the message `Ping Apps` has a parameter `appName`. The description of these parameter is shown as below.
+
+<b>Table 1.</b> Additional Frame Info Definitions on Protocol spec
+
+| FRAME INFO VALUE | NAME | DESCRIPTION |
+| :- | :- | :- |
+| 0x0a | Request Additional Support | Requests that the additional supports that SDL app needs. |
+| 0x0b | Ping Apps | Acknowledges that SDL app is available the additional supports. |
```

Comment:
Can you please reword or provide clarification for this description? I'm unsure of its intended meaning.

Reply:
What you mean is that you cannot understand the DESCRIPTION of `Request Additional Support` and `Ping Apps` ?
If correct, we will modify the description of `Request Additional Support` to "Requests the required app support" or "Notifies the required app support".
Also, we will modify the description of `Ping Apps` to "Acknowledges the required app support is available" or "Notifies the required app support".
Which description is better?
If our understanding is wrong, could you please teach us the details?

Reply:
Thank you for the additional information. Please update the `Ping Apps` description to "Notifies the required app of the support available." I've left a separate comment regarding the description for `Request Additional Support`.

Reply:
@theresalech -san,
We modified to "Notifies the required app of the support available." and made a commit it.
Thank you.